Syntax: constant operators, assembler instruction

QSmally opened the pull request:
* adds constant operators (`.label & 0x1F`, `.label + 8`, `64 mod .label lsh 1`);
* adds `BREAKPOINT` assembler instruction.

Commit:
Syntax: add constant operators

Commit:
Syntax: `BREAKPOINT` alias/assembler instruction

RedCMD commented:
`punctuation.definition.tag` is a gray colour
`keyword.operator` is white
Are you sure you want to assign `operators` that colour?

RedCMD committed:
4 wide tabs

QSmally commented:
Tbh, @RedCMD, anything that looks good:

```
MLI zer, .string_label & 0x1F
MLI zer, .string_label and 0x1F
```

RedCMD committed:
`punctuation.definition.tag` => `keyword.operator`

Changed `#operators` from `punctuation.definition.tag`(gray) to `keyword.operator`(while)

QSmally commented:
LGTM
